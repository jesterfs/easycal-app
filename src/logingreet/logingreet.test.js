import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import LoginGreet from './logingreet'
import renderer from 'react-test-renderer';

describe('LoginGreet component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <LoginGreet />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
  
  it('renders the UI as expected', () => {
    const div = document.createElement('div')
    const tree = renderer
      .create(<BrowserRouter>
        <LoginGreet />
      </BrowserRouter>,
      div)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

});