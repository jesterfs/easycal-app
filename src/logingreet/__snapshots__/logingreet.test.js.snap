// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginGreet component renders the UI as expected 1`] = `
<div
  className="LoginGreet, greetgroup"
>
  <div
    className="item"
  >
    <h2>
      Welcome Back!
    </h2>
  </div>
  <div
    className="item"
  >
    <form
      className="login-form"
      onSubmit={[Function]}
    >
      <div>
        <label
          htmlFor="email"
        >
          Email
        </label>
        <input
          id="email"
          name="email"
          type="email"
        />
      </div>
      <div>
        <label
          htmlFor="password"
        >
          Password
        </label>
        <input
          id="password"
          name="password"
          type="password"
        />
      </div>
      <button
        type="submit"
      >
        Login
      </button>
    </form>
  </div>
</div>
`;
